nativelib/src/main/scala/scala/scalanative/native/complex.scala

```diff
@@ -0,0 +1,249 @@
+  @name("scalanative_cexp")
+  def cexp(complex: Ptr[CDoubleComplex],
+           result: Ptr[CDoubleComplex]): Ptr[CDoubleComplex] = extern
+  //  extern long double complex cexpl(long double complex);
+
+  @name("scalanative_clogf")
+  def clogf(complex: Ptr[CFloatComplex],
+            result: Ptr[CFloatComplex]): Ptr[CFloatComplex] = extern
+  @name("scalanative_clog")
+  def clog(complex: Ptr[CDoubleComplex],
+           result: Ptr[CDoubleComplex]): Ptr[CDoubleComplex] = extern
+  //  extern long double complex clogl(long double complex);
+
+  @name("scalanative_cabsf")
+  def cabsf(complex: Ptr[CFloatComplex]): CFloat = extern
+  @name("scalanative_cabs")
+  def cabs(complex: Ptr[CDoubleComplex]): CDouble = extern
+  //  extern long double cabsl(long double complex);
+
+  @name("scalanative_cpowf")
+  def cpowf(x: Ptr[CFloatComplex],
+            y: Ptr[CFloatComplex],
+            result: Ptr[CFloatComplex]): Ptr[CFloatComplex] = extern
+  @name("scalanative_cpow")
+  def cpow(x: Ptr[CDoubleComplex],
+           y: Ptr[CDoubleComplex],
+           result: Ptr[CDoubleComplex]): Ptr[CDoubleComplex] = extern
+  //  extern long double complex cpowl(long double complex, long double complex);
+
+  @name("scalanative_csqrtf")
+  def csqrtf(complex: Ptr[CFloatComplex],
+             result: Ptr[CFloatComplex]): Ptr[CFloatComplex] = extern
+  @name("scalanative_csqrt")
+  def csqrt(complex: Ptr[CDoubleComplex],
+            result: Ptr[CDoubleComplex]): Ptr[CDoubleComplex] = extern
+  //  extern long double complex csqrtl(long double complex);
+
+  @name("scalanative_cargf")
+  def cargf(complex: Ptr[CFloatComplex]): CFloat = extern
+  @name("scalanative_carg")
+  def carg(complex: Ptr[CDoubleComplex]): CDouble = extern
+  //  extern long double cargl(long double complex);
+
+  @name("scalanative_cimagf")
+  def cimagf(complex: Ptr[CFloatComplex]): CFloat = extern
+  @name("scalanative_cimag")
+  def cimag(complex: Ptr[CDoubleComplex]): CDouble = extern
+  //  extern long double cimagl(long double complex);
+
+  @name("scalanative_conjf")
+  def conjf(complex: Ptr[CFloatComplex],
+            result: Ptr[CFloatComplex]): Ptr[CFloatComplex] = extern
+  @name("scalanative_conj")
+  def conj(complex: Ptr[CDoubleComplex],
+           result: Ptr[CDoubleComplex]): Ptr[CDoubleComplex] = extern
+  //  extern long double complex conjl(long double complex);
+
+  @name("scalanative_cprojf")
+  def cprojf(complex: Ptr[CFloatComplex],
+             result: Ptr[CFloatComplex]): Ptr[CFloatComplex] = extern
+  @name("scalanative_cproj")
+  def cproj(complex: Ptr[CDoubleComplex],
+            result: Ptr[CDoubleComplex]): Ptr[CDoubleComplex] = extern
+  //  extern long double complex cprojl(long double complex);
+
+  @name("scalanative_crealf")
+  def crealf(complex: Ptr[CFloatComplex]): CFloat = extern
+  @name("scalanative_creal")
+  def creal(complex: Ptr[CDoubleComplex]): CDouble = extern
+  //  extern long double creall(long double complex);
```

Comment:
Ok, this looks much better and points to #27. Note to self - remember to look at current issues first.